Makefile

```diff
@@ -106,7 +108,7 @@ check: ocaml_checks libp2p_helper ## Check that all OCaml packages build without
 	dune build @src/check
 
 .PHONY: build
-build: ocaml_checks reformat-diff libp2p_helper ## Build the main project executables
+build: libp2p_helper ## Build the main project executables
```

Comment:
This is something I'm not sure. ocaml_checks triggers ocaml switch update, which in my case rebuild entire env from scratch, which takes forever. 

Detaching ocaml_checks spares this burden